src/ChannelServer/Database/ChannelDb.cs

```diff
@@ -110,8 +110,8 @@ public Character GetCharacter(long accountId, long characterId)
 					character.MaxExp = reader.GetInt32("maxExp");
 					character.Hp = reader.GetInt32("hp");
 					character.MaxHp = reader.GetInt32("maxHp");
-					character.Sp = reader.GetInt32("sp");
-					character.MaxSp = reader.GetInt32("maxSp");
+					character.Sp = reader.GetInt16("sp");
+					character.MaxSp = reader.GetInt16("maxSp");
```

Comment:
Should we change these columns to be `smallint`?

Reply:
Actually, in C# it's customary to use int for pretty much anything, unless you need specific types, like for inter-operation. One reason for this is that .NET is optimized for working with ints. In this case an int also ensures that we won't have to change anything but the put call, should the type ever change. That's why they are all int, even if the packet uses a short.

Initially I did the same thing in Aura, using the exact types for everything, thinking that it would better explain what values a property can take, but for me personally it turned out that I never cared one bit about what the types actually are, because the properties need to check and clamp the incoming values anyway. Enforcing the actual types made our lives harder though, because you constantly have to figure out the type, cast, check values, etc.

Though I just noticed that our PutShort has an int overload... I don't remember why I did that, that seems unsafe and should probably be changed =P"

Reply:
Yes, that was a good explanation, and if .NET is optimized for dealing with `int` then we should take advantage of that. 

Now thinking about this, it occurred to me that if for some reason a value larger than 32767 existed in the database, placing that value as short would mean rollover. It may not be an issue if only the application stores the value, but if someone were to change values on their own in the database, they would need to be aware of the size limitations enforced by the application.